bundles/c4c-mock-0.1.0/docs/overview.md

```diff
@@ -0,0 +1,6 @@
+---
+title: SAP C4C Mock
+type: Details
+---
+Use the c4c-mock application to mock an **SAP C4C** installation. The application is based on the [**Varkes**](https://github.com/kyma-incubator/varkes) project and is part of the [**XF-Application-Mocks**](https://github.com/SAP/xf-application-mocks).
+After creating service instance, a new navigation node appears in the Kyma Console. Use the functionality it provides to connect the Mock to your Kyma cluster and browse the APIs.
```

Comment:
As the console integration is not done yet, please write how to access the UI: https://c4c-<namespace>.<cluster_domain>